src/notation/view/abstractnotationpaintview.h

```diff
@@ -235,13 +232,11 @@ protected slots:
 
     bool ensureViewportInsideScrollableArea();
 
-    muse::RectF scrollableAreaRect() const;
-
     qreal horizontalScrollableSize() const;
     qreal verticalScrollableSize() const;
 
-    bool adjustCanvasPosition(const muse::RectF& logicRect, bool adjustVertically = true);
-    bool adjustCanvasPositionSmoothPan(const muse::RectF& cursorRect);
+    bool adjustCanvasPosition(const muse::RectF& logicRect, bool adjustVertically = true, bool userTriggeredMove = true);
+    bool adjustCanvasPositionSmoothPan(const muse::RectF& cursorRect, bool userTriggeredMove = true);
```

Comment:
It looks like this new `userTriggeredMove` parameter is not used anywhere, or is it? If indeed not, I would suggest not adding it.